hexrd/ui/interactive_template.py

```diff
@@ -8,17 +8,13 @@
 
 from skimage.draw import polygon
 
-from hexrd.ui.create_hedm_instrument import create_hedm_instrument
-from hexrd.ui import resource_loader
 from hexrd.ui.hexrd_config import HexrdConfig
 from hexrd.ui.utils import has_nan
 
 
 class InteractiveTemplate:
-    def __init__(self, parent=None):
-        self.parent = parent.image_tab_widget.image_canvases[0]
-        self.ax = self.parent.axes_images[0]
-        self.panels = create_hedm_instrument().detectors
+    def __init__(self, parent, detector, instrument=None):
+        self.image_tab_widget = parent.image_tab_widget
```

Comment:
If `parent` isn't actually used for anything, then it might be better for us to just change that argument to `image_tab_widget` instead, and pass the `image_tab_widget`.

Reply:
This is actually removed altogether with [8ebecb9](https://github.com/HEXRD/hexrdgui/pull/1569/commits/8ebecb923eb7a5016946d2aab5ea2e43561a3bc3) now because we are using signals instead of passing around the main window (like we did with the `HandDrawnMaskDialog`)